fast-reid/tools/train_net.py

```diff
@@ -8,12 +8,26 @@
 import sys
 
 sys.path.append('.')
+sys.path.append('fast-reid')
 
 from fastreid.config import get_cfg
 from fastreid.engine import DefaultTrainer, default_argument_parser, default_setup, launch
 from fastreid.utils.checkpoint import Checkpointer
 
 
+'''
+USAGE:
+need to be under the /fast-reid folder and then run:
+python3 tools/train_net.py --config-file ./configs/DukeMTMC/bagtricks_R101-ibn.yml MODEL.WEIGHTS 
+checkpoints/duke_bot_R101-ibn-new.pth MODEL.DEVICE "cuda:0" DATASETS.DATASET "DukeMTMC"
```

Comment:
This is subject to change later on when we we'll work with multiple GPU's, other configs / weights. 
For now maybe write "sample usage" ---> ...

Reply:
this is only an example of how to run it so we know which arguments to pass. If later the running mechanism changes we will have to also change the code